storage/api/Storage/BucketSetPublicIam.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_set_bucket_public_iam]
+
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class BucketSetPublicIam
+{
+	public void SetPublicAccess(string bucketName)
+    {
+        var role = "roles/storage.objectViewer";
+
+        var storage = StorageClient.Create();
+
+		var bucketPolicy = storage.GetBucketIamPolicy(bucketName);
```

Comment:
format this properly and make sure in all file code should be well formatted you can run `ctrl+k+d` for same.